cmd/skuba/cluster/init.go

```diff
@@ -54,7 +56,15 @@ func NewInitCmd() *cobra.Command {
 				klog.Fatalf("init failed due to error: %s", err)
 			}
 		},
-		Args: cobra.ExactArgs(1),
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return fmt.Errorf("accepts 1 arg, received %d", len(args))
+			}
+			if len(args[0]) == 0 {
+				return fmt.Errorf("Invalid cluster name: expected non empty cluster name")
+			}
```

Comment:
```suggestion
			if len(args[0]) == 0 {
				return fmt.Errorf("Invalid cluster name: expected non empty cluster name")
			}
			if len(args) != 1 {
				return fmt.Errorf("accepts 1 arg, received %d", len(args))
			}
```

Reply:
If you validate in this order you will get a panic if args in empty

Reply:
Thanks, make some change~

Reply:
@jenting @chentex meant that it will panic if in the wrong order. Anyway it won't cover `cluster init "" --control-plane 127.0.0.1`. It's the length of the argument that can't be empty, not the amount of arguments. The amount of arguments is already taken care of by len(args).